docs/source/10_deployment/07_aws_batch.md

```diff
@@ -300,16 +300,9 @@ aws_batch:
 You're nearly there! Before you can use the new runner, you need to add a `cli.py` file at the same level as `settings.py`, using [the template we provide](../07_extend_kedro/01_common_use_cases.md#use-case-3-how-to-add-or-modify-cli-commands). Add the following `run()` function to your `cli.py` file to make sure the runner class is instantiated correctly:
 
 ```python
-def run(tag, env, parallel, ...):
+def run(tag, env, ...):
     """Run the pipeline."""
-    if parallel and runner:
-        raise KedroCliError(
-            "Both --parallel and --runner options cannot be used together. "
-            "Please use either --parallel or --runner."
-        )
-    runner = runner or "SequentialRunner"
-    if parallel:
-        runner = "ParallelRunner"
+    runner = load_obj(runner or "SequentialRunner", "kedro.runner")
```

Comment:
I think `runner` need to be a `str` still - doesn't the object get loaded in `_instantiate_runner` below?

Reply:
Very good point! Thank you 👍

Reply:
```suggestion
    runner = runner or "SequentialRunner"
```

Reply:
Very good point, thank you!
```suggestion
    runner = runner or "SequentialRunner"
```